packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -78,30 +78,41 @@ export class FauxClassGenerator {
     return Promise.resolve('');
   }
 
-  private generateChildRelationship(rel: ChildRelationship): string {
-    if (rel.relationshipName) {
-      return `List<${rel.childSObject}> ${rel.relationshipName}`;
+  private stripId(name: string): string {
+    if (name.endsWith('Id')) {
+      return name.slice(0, name.length - 2);
     } else {
-      // expect the name to end with Id, then strip off Id
-      if (rel.field.endsWith('Id')) {
-        const nameWithoutId = rel.field.slice(0, rel.field.length - 2);
-        return `${rel.childSObject} ${nameWithoutId}`;
-      } else {
-        return '';
-      }
+      return name;
     }
   }
 
+  private getReferenceName(relationshipName: string, name: string): string {
+    if (relationshipName) {
```

Comment:
`return relationshipName ? relationshipName : this.stripId(name);`

Reply:
yep, that is better - I always find that harder to read so I default to an "if" unless I think to do it